### ISSUE TITLE
Some items on Your Submissions page are missing mouse hover focus indicators

### ISSUE THREAD

**Issue by ctrance** [ucla-oarc-mobile/casa-on-rails#210]
@krozett The following are missing mouse hover focus indicators on Your Submissions page:
a.	Show entries dropdown list
![image](https://user-images.githubusercontent.com/1034174/108117766-385a4000-7052-11eb-8786-d624b767b4eb.png)
b.	Table header sort buttons
c.	Show and Edit links
![image](https://user-images.githubusercontent.com/1034174/108117782-3db78a80-7052-11eb-9c01-9971e04c179f.png)

Accessibility success criterion:
* https://www.w3.org/TR/WCAG21/#use-of-color
* https://www.w3.org/TR/WCAG21/#non-text-contrast

**Comment by ctrance** [comment]
Closing old tickets that we will not fix for now.
